### ISSUE TITLE
Feedback - Week 5 Assignment

### ISSUE THREAD

**Issue by ntuvera** [jimmylado/TriviaGame#1]
Hey @jimmylado, I cloned your repo and tried out your trivia game...

Great job on the functionality.  You've met all the requirements and gave it a style all your own.

Looking through your code, it all works which is great.  But I noticed a few things that caught my eye.  I see in your question objects you're retyping the string for the "correct answer", why repeat yourself, when you can refer back to the object property then the array index?  The way you have your objects in an array does make abstracting the reference to James harder.  

If the question1 was its own object, you could do something like ```question1: { answer : question1['choices'][0]}``` it might not work in this specific instance.  But the point is to avoid repeating yourself.  Why type James twice, when you reference back to your array of choices and select the correct index?

I like your use of jquery to append and remove your divs.  Your code is clear and concise.  You've got good use of setTimeouts, you'd be surprised how often people use setInteval instead and cause setInterval hell for themselves.  Your use of object functions and properties is great.  What do you think about object oriented programming?  

Keep up the good work and get back to me on Slack or in person.

**Comment by jimmylado** [comment]
Yeah you're right I thought it would be easier to latch to that correct answer variable instead of getting it otherwise. I could have done it and been cleaner for sure.

Objects definitely make things easier some things are a little confusing especially the things we were going over today when selecting particular html divs and placing objects into those specific divs can get a bit complicated when it comes to multiple objects. Overall though I feel that's something trial and error is making me more solid at !
